science/pymol/Portfile

```diff
@@ -83,7 +101,7 @@ use_parallel_build yes
 
 build {}
 
-destroot.cmd ${python.bin} setup.py --no-user-cfg --osx-frameworks --glut --use-msgpackc=no
+destroot.cmd ${python.bin} setup.py --no-user-cfg --glut --use-msgpackc=no
```

Comment:
as far as I can tell you need the `--glut` and `no-osx-frameworks` only for the `x11` variant not for Qt.